### PR TITLE
Orca: Better loss support, pack outputs anyhow

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

https://github.com/analytics-zoo/Telecom-CTC-E2E/issues/19

Assumption: Loss_func must absorb tensors **one by one**, targets(a tensor or a list of tensors) are placed at the last of batch in dataloader.

### CODE REVIEW

File: python/orca/src/bigdl/orca/learn/pytorch/pytorch_metrics.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self):
 
     def __call__(self, preds, targets):
         preds, target = _unify_input_formats(preds, targets)
```
[the remark this answers is not in the document]
Comment (leonardozcm) [reply python/orca/src/bigdl/orca/learn/pytorch/pytorch_metrics.py:74]: > why not change target to targets here? Weird for preds and target...

I agree, this is a typo I found and fixed by the way.

[the remark this answers is not in the document]
Comment (hkvision) [reply python/orca/src/bigdl/orca/learn/pytorch/pytorch_metrics.py:74]: raise another PR to modify this?
